README.md

```diff
@@ -40,364 +43,346 @@ some kind of discriminant:
-  slot of `E`.
-- `Enum.values(E)` - Returns an `Iterator` for the value on `E` of each member in the 
-  \[\[EnumMembers]] internal slot of `E`.
-- `Enum.entries(E)` - Returns an `Iterator` for each member in the \[\[EnumMembers]] internal
-  slot of `E`, where each result is two-element array containing the enum member name at index 0 
-  and the enum member value at index 1.
-- `Enum.has(E, key)` - Returns `true` if the the \[\[EnumMembers]] internal slot of `E` contains `key`.
-- `Enum.hasValue(E, value)` - Returns `true` if the \[\[EnumMembers]] internal slot of `E` contains a
-  member whose value on `E` corresponds to `value`.
-- `Enum.getName(E, value)` - Gets the first name in the \[\[EnumMembers]] internal slot of `E` whose
-  value on `E` corresponds to `value`.
-- `Enum.format(E, value)` - Calls the `@@formatEnum` method of `E` with argument `value`.
-- `Enum.parse(E, value)` - Calls the `@@parseEnum` method of `E` with argument `value`.
-- `Enum.create(members)` - Creates an _enum object_ using the property keys and values of `members`
-  as the _enum members_ for the new enum.
-- `Enum.flags(descriptor)` - A built-in decorator that modifies the _enum object_ in the following ways:
-  - The auto-increment behavior is changed to shift the current auto-increment value left by 1. 
-  - The `@@parseEnum` method is modified to parse a comma-separated string and OR the resulting values
-    together. If no corresponding name can be found and the name can be successfully coerced to a number,
-    that number is OR'ed with the result.
-  - The `@@formatEnum` method is modified to convert a bitwise combination of flag values into a comma
-    separated string of corresponding names. If no corresponding name can be found, the SV of the 
-    bits is appended to the string.
+TypeScript's `enum` supports auto-initialization of enum members:
 
 ```ts
-let Enum: {
-  keys(E: object): IterableIterator<string | symbol>;
-  values(E: object): IterableIterator<any>;
-  entries(E: object): IterableIterator<[string | symbol, any]>;
-  has(E: object, key: string | symbol): boolean;
-  hasValue(E: object, value: any): boolean;
-  getName(E: object, value: any): string | undefined;
-  format(E: object, value: any): string | symbol | undefined;
-  parse(E: object, value: string): any;
-  create(members: object): object;
-  flags(descriptor: EnumDescriptor): EnumDescriptor;
-};
+enum Numbers {
+  zero, // 0
+  one,  // 1
+  two   // 2
+}
 ```
 
-<!--
-# Grammar
+However, this behavior is contentious amongst some TC39 delegates and has been removed from this proposal. TypeScript
+will continue to support auto-initializiation due to its frequent use within the developer community, but would emit
+explicit initializers to JavaScript. It is possible that another form of auto-initializiation may be introduced in the
+future that could be utilized by both TypeScript and ECMAScript.
```

Comment:
This sounds ominously vague until you read further down the doc at which point it becomes concrete.

```suggestion
future that could be utilized by both TypeScript and ECMAScript.  Please see the Future Directions section on Auto-Initializers for examples of syntax that could achieve this. 
```

(please linkify)